### PR TITLE
Add prettier and vue-router to coding test

### PR DESCRIPTION
**Pull request by glounder**

### CODE REVIEW

**Review by coleman-benja — APPROVED** [review]

**Review by chrisdail — APPROVED** [review]

**Review by pchisholm — APPROVED** [review]

File: src/router.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  { name: 'Home', path: '', redirect: { name: 'Listings' } },
+  { name: 'Login', path: '/login', component: Login },
+  {
+    name: 'Listings',
+    path: '/listings',
+    component: Listings,
+  },
+  {
+    name: 'BookListing',
+    path: '/book',
```
Comment (pmdarrow) [inline src/router.js]: Should this be `/listings/<id>`?

File: src/router.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    component: BookListing,
+  },
+  { name: 'MyBookings', path: '/bookings/my', component: MyBookings },
```
Comment (pmdarrow) [inline src/router.js]: What do you think about simplifying this to `/bookings`?

File: src/router.js (hunk 1 above)
Comment (glounder) [reply src/router.js]: Done

File: src/router.js (hunk 2 above)
Comment (glounder) [reply src/router.js]: Done

**Pull request merged**